typescript/infra/scripts/agent-utils.ts

```diff
@@ -113,6 +114,14 @@ export function withKeyRoleAndChain<T>(args: yargs.Argv<T>) {
     .alias('i', 'index');
 }
 
+// missing chains are chains needed which are not as part of defaultMultisigConfigs in sdk/src/consts/ but are in chainMetadata
+export function withMissingChains<T>(args: yargs.Argv<T>) {
+  return args
+    .describe('new-chains', 'new chains to add')
+    .string('new-chains')
+    .alias('n', 'new-chains');
+}
```

Comment:
this is oddly specific
doesnt chainMetadata also require SDK changes?

Reply:
oddly specific how, you want to create keys for new chains which I don't have in the SDK so I'll ask for the threshold and total # of keys.

Reply:
updated to the total number of keys